package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mobx",
-  "version": "4.0.0-beta.4",
+  "version": "15.0.0-alpha.1",
```

Comment:
Do you mean "5.0.0-alpha.1"? :-)

Reply:
Actually might be 15 :-) .Completely in the spirit of React.

But more importantly, I want to keep supporting MobX 4 for those we have to support browsers without proxies. And calling this one 5 would hamper the ability to make breaking changes on the "4-branch"

Reply:
hmhm. Major is the new minor

Reply:
I mean i have no objection, it's just a bit mind bending ^^